docs/CSharp_API.md

```diff
@@ -2,7 +2,7 @@
 The ONNX runtime provides a C# .Net binding for running inference on ONNX models in any of the .Net standard platforms. The API is .Net standard 1.1 compliant for maximum portability. This document describes the API. 
 
 ## NuGet Package
-The Microsoft.ML.OnnxRuntime Nuget package includes the precompiled binaries for ONNX runtime, and includes libraries for Windows 10 platform and X64 CPUs. The APIs conform to .Net Standard 1.1.
+The Microsoft.ML.OnnxRuntime Nuget package includes the precompiled binaries for ONNX runtime, and includes libraries for Windows and Linux platforms with X64 CPUs. The APIs conform to .Net Standard 1.1.
```

Comment:
Specify Ubuntu

Reply:
we would support generic Linux (the distrib binary). It may be mentioned that it is tested on Ubuntu 16.04